<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSceneAddTable</class>
 <widget class="QDialog" name="CSceneAddTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工况添加</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>工况名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>厂    站：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="editStation"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>设    备：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="editDev"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>默认画面：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editFileName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>备    注：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="editRemark"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonEnter">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editFileName</tabstop>
  <tabstop>editRemark</tabstop>
  <tabstop>buttonApply</tabstop>
  <tabstop>buttonEnter</tabstop>
  <tabstop>buttonExit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>CSceneAddTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
